domain/src/main/java/no/difi/meldingsutveksling/arkivmelding/JournalposttypeMapper.java

```diff
@@ -0,0 +1,37 @@
+package no.difi.meldingsutveksling.arkivmelding;
+
+import com.google.common.collect.HashBiMap;
+import lombok.extern.slf4j.Slf4j;
+import no.arkivverket.standarder.noark5.metadatakatalog.Journalposttype;
+
+@Slf4j
+public class JournalposttypeMapper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)